.gitignore

```diff
@@ -61,6 +61,7 @@ typings/
 # Compiled files
 src/ts/*.js*
 src/ts/**/*.js*
+src/js/turf-bundle.js
```

Comment:
this is intermediate file generated by `npm run turf-build` no need to be git tracked